scala-interpreter/build.sbt

```diff
@@ -18,3 +18,4 @@ import sbt.Tests.{Group, SubProcess}
  */
 
 libraryDependencies ++= Dependencies.sparkAll.value
+libraryDependencies += "com.github.jupyter" % "jvm-repr" % "0.1.0"
```

Comment:
I think we'll need to take a look after this PR at the other interpreters. PySpark, for example, should adhere to the common python convention to use _repr_html_ for objects that exist in the python process, and should use the Jupyter library for objects that are resident in the JVM (like DataFrames).

The background on Jupyter's jvm-repr library is that I'm trying to work with the Jupyter community to standardize a system for libraries and kernels to collaborate on rich representations for the JVM. Vegas, for example, should register a function to display its objects so it doesn't have to be built into the kernel or bolted on by users.